faker/providers/ssn/lv_LV/__init__.py

```diff
@@ -1,7 +1,57 @@
-from .. import Provider as BaseProvider
+import datetime
 
+from .. import Provider as SsnProvider
+
+
+class Provider(SsnProvider):
+    def ssn(self, min_age: int = 0, max_age: int = 105) -> str:
+        """
+        Returns 11 character Latvian personal identity code (Personas kods).
+        This function assigns random age to person.
+
+        Personal code consists of eleven characters of the form DDMMYYCZZZQ, where
+        DDMMYY is the date of birth, C the century sign, ZZZ the individual
+        number and Q the control character (checksum). The number for the
+        century is either 0 (1800–1899), 1 (1900–1999), or 2 (2000–2099).
+        """
+
+        def _checksum(ssn_without_checksum):
+            weights = [1, 6, 3, 7, 9, 10, 5, 8, 4, 2]
+            weighted_sum = sum(int(digit) * weight for digit, weight in zip(ssn_without_checksum, weights))
+            print(weighted_sum)
```

Comment:
This `print` statement seems to have been committed accidentally.

Reply:
See it fixed.